style.css

```diff
@@ -0,0 +1,166 @@
+.works {
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  margin: 30px;
+  row-gap: 30px;
+}
+
+.text h2 {
+  font-family: Poppins, sans-serif;
+  font-weight: bold;
+  font-size: 35px;
+  color: rgb(23, 43, 77);
+  line-height: 40px;
+  display: flex;
+  margin-top: 5px;
+  margin-bottom: -5px;
+}
+
+.list ul {
+  display: flex;
+  margin-left: -40px;
+  align-items: flex-start;
+}
+
+li {
+  list-style-type: none;
+  padding: 5px;
+  color: rgb(172, 179, 192);
+  font-family: Poppins, sans-serif;
+  font-weight: bold;
+  font-size: 12px;
+}
+
+.CANOPY {
+  color: rgb(52, 69, 99);
+}
+
+.text p {
+  font-family: Poppins, sans-serif;
+  font-weight: normal;
+  font-size: 17px;
+  line-height: 24px;
+  color: rgb(52, 69, 99);
+  display: flex;
+  justify-content: flex-start;
+  width: 300px;
+}
+
+.button-list ul {
+  display: flex;
+  margin-left: -40px;
+  align-items: flex-start;
+}
+
+.button-list ul li {
+  color: rgb(96, 112, 255);
+  padding: 5px 15px;
+  border-radius: 6px;
+  background-color: rgb(235, 235, 255);
+  margin-right: 13px;
+}
+
+a {
+  color: rgb(96, 112, 255);
+  text-decoration: none;
+  font-weight: bold;
+  font-size: 13px;
+  font-family: Poppins, sans-serif;
+}
+
+button {
+  padding: 10px 20px;
+  text-align: center;
+  text-decoration: none;
+  font-weight: bold;
+  font-size: 16px;
+  margin: 4px 2px;
+  cursor: pointer;
+}
+
+.btn {
+  background-color: white;
+  border: 1.6px solid rgb(96, 112, 255);
+  border-radius: 8px;
+  color: rgb(96, 112, 255);
+}
```

Comment:
- [ ] [OPTIONAL] Your buttons looks great. In addition to what you have done, I would suggest you to implemented button interaction styles. When you hover on the button, it will look different than when you remove the mouse from it. These are the kind of interaction styles I am talking about. You may refer to this guide given in the Figma design 👇 
![image](https://user-images.githubusercontent.com/90222110/178072459-0a152735-28b8-4c7c-8085-f44c5f66a5ff.png)